src/Surfnet/StepupSelfService/SelfServiceBundle/Command/RemoteVetValidationCommand.php

```diff
@@ -18,21 +18,12 @@
 
 namespace Surfnet\StepupSelfService\SelfServiceBundle\Command;
 
-use Surfnet\StepupMiddlewareClientBundle\Identity\Dto\Identity;
-use Surfnet\StepupMiddlewareClientBundle\Identity\Dto\VerifiedSecondFactor;
 use Surfnet\StepupSelfService\SelfServiceBundle\Service\RemoteVetting\Value\AttributeMatchCollection;
-use Surfnet\StepupSelfService\SelfServiceBundle\Service\RemoteVetting\Value\ProcessId;
-use Surfnet\StepupSelfService\SelfServiceBundle\Service\RemoteVetting\Value\AttributeMatch;
 
 class RemoteVetValidationCommand
 {
     /**
-     * The attributes and their state
-     *
-     *  [key => [
-     *      'valid' => true,
-     *      'remarks' => 'remarks',
-     *  ]]
+     * The attribute matches
```

Comment:
This is not that useful to me as a reader it's not entirely clear what the purpose of this field is. My suggestion is to either remove the doc block, or add additional explanation.

Reply:
Removed it.